gap/perm/giant.gi

```diff
@@ -548,9 +546,7 @@ end;
 
 
 ##  SLP for pi from (1,2,3), sigma
-
 RECOG.SLPforAn :=  function( n, pi )
-
```

Comment:
Might want to drop the diffs in giant.gi?

Reply:
Yepp